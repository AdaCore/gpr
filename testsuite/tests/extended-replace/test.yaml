description:
    Simple test for extended projects (with replaced unit)
driver: build_and_run
project_file: extended_replace
main: main
