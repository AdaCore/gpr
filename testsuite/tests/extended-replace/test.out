**************** Iterator Prj1
Prj1     K_STANDARD
 > p1/pck1.adb    language: Ada    Kind: S_BODY   unit: PCK1
 > p1/pck1.ads    language: Ada    Kind: S_SPEC   unit: PCK1
**************** Iterator Prj2
Prj1     K_STANDARD
 > p1/pck1.adb    language: Ada    Kind: S_BODY   unit: PCK1
 > p1/pck1.ads    language: Ada    Kind: S_SPEC   unit: PCK1
Prj2     K_STANDARD
 > p2/pck1.adb    language: Ada    Kind: S_BODY   unit: PCK1
 > p1/pck1.ads    language: Ada    Kind: S_SPEC   unit: PCK1
 > p2/pck2.ads    language: Ada    Kind: S_SPEC   unit: PCK2
